songbird/exceptions.py

```diff
@@ -1,16 +1,23 @@
 from .songbird import (
-    YtdlError, FfmpegError, SongbirdError, ConsumedSourceError, CouldNotOpenFileError,
-    UseAsyncConstructorError
+    YtdlError,
+    FfmpegError,
+    SongbirdError,
+    ConsumedSourceError,
+    CouldNotOpenFileError,
+    UseAsyncConstructorError,
 )
 
-__all__ = (
+
+class QueueError(SongbirdError):
+    """Queue raises an exception"""
+
+
+__all__ = [
```

Comment:
I use () for `__all__` in this project

Reply:
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package
Its list

Reply:
Pep8: https://www.python.org/dev/peps/pep-0008/#id20